packages/iTwinUI-react/src/core/Table/hooks/useResizeColumns.tsx

```diff
@@ -113,14 +111,18 @@ const defaultGetResizerProps = (ownerDocument: Document | undefined) => (
         moveHandler: (e: MouseEvent) => dispatchMove(e.clientX),
         upEvent: 'mouseup',
         upHandler: () => {
-          ownerDocument.removeEventListener(
+          ownerDocument.current?.removeEventListener(
             'mousemove',
             handlersAndEvents.mouse.moveHandler,
           );
-          ownerDocument.removeEventListener(
+          ownerDocument.current?.removeEventListener(
             'mouseup',
             handlersAndEvents.mouse.upHandler,
           );
+          ownerDocument.current?.removeEventListener(
```

Comment:
I saw that it was triggering on resize end every time mouse leaves the screen. Now it only triggers that once.